app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
```

Comment:
* Due to the changes made in respect of the previous reviewer's request your code now has a bug, as when i try signing up i get an error, Kindly fix this. below  is a screenshot of the said error i get ⬇️ 
![image](https://user-images.githubusercontent.com/52497246/180168662-417091ef-ed61-493a-a39c-c497457aa06a.png)